.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dh-moduleRef-siblings-195
+  pull_request:
+    branches:
+      - dh-moduleRef-siblings-195
```

Comment:
I think this means the branch into which the PR shall be merged, so it should be `master`.

Reply:
> I think this means the branch into which the PR shall be merged, so it should be `master`.

I just for testing now, was not sure if it works at all. Will change both to master before we can merge